src/main/java/view/Applyview.java

```diff
@@ -146,7 +146,7 @@ public void propertyChange(PropertyChangeEvent evt) {
         if (y.equals("success")) {
             // Results View Not implemented yet, will be implemented soon.
             ApplyState state = (ApplyState) evt.getNewValue();
-            JOptionPane.showMessageDialog(this, state.getUni());
+            JOptionPane.showMessageDialog(this, state.getUni().getSchoolName() + " SAT:"+ state.getUni().getAverageSATScore()+ " ACT:"+ state.getUni().getAverageACTScore());
```

Comment:
Preferably, returning the university with all of the parameters out should be better for the user, but at least a url should be shown.